app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -55,6 +55,10 @@ const tabList = [
 function annotHasDe(exploreInfo, exploreParams) {
   const flags = getFeatureFlagsWithDefaults()
   if (!flags?.differential_expression_frontend || !exploreInfo) {
+    // set isDifferentialExpressionEnabled to false as user cannot see DE results, even if present for annotation
+    if (window.SCP) {
+      window.SCP.isDEEnabled = false
```

Comment:
Blocker: straggler

```suggestion
      window.SCP.isDifferentialExpressionEnabled = false
```

Reply:
Ack I had changed that and apparently did a command-Z before committing... thanks for finding that.

Reply:
[c8b8588](https://github.com/broadinstitute/single_cell_portal_core/pull/1695/commits/c8b858818738b41515d2995e2a5d6fcb4ab2591f)